docs/source/models/intervention_models/mncnh_pregnancy/iv_iron_antenatal/iv_iron_mncnh.rst

```diff
@@ -0,0 +1,214 @@
+.. _intervention_iv_iron_antenatal_mncnh:
+
+=====================================================
+Antenatal Intravenous Iron - MNCNH Portfolio Model
+=====================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - IV
+    - Intravenous
+    - 
+
+Intervention Overview
+-----------------------
+
+The antenatal IV iron intervention is intended to treat moderate and severe iron-deficiency anemia in pregnancy. It is administered those in their 2nd or 3rd trimester of pregnancy who have a hemoglobin level less than 100 grams per liter and low ferritin levels.
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+.. todo::
+
+  Update baseline coverage data 
+
+Vivarium Modeling Strategy
+--------------------------
+
+The IV iron intervention is included in the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>` of in the pregnancy component of the :ref:`MNCNH porfolio simulation <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+We will model the following eligibility for the antenatal IV iron intervention:
+
+#. Attends later prenancy ANC visit (according to the :ref:`antenatal care attendance module <2024_vivarium_mncnh_portfolio_anc_module>`)
+#. Test hemoglobin exposure of less than 100 grams per liter (according to the :ref:`anemia screening intervention <anemia_screening>` result tracked in the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`)
+#. Tests low ferritin exposure (according to the :ref:`anemia screening intervention <anemia_screening>` result tracked in the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`)
```

Comment:
"Test" above vs "Tests" here---both make sense, but we should be consistent.  Maybe there is an even clearer way to say it in both places...

Reply:
Updated to "low hemoglobin/ferritin according to the hemoglobin/ferritin screening test." Thanks